doc/readme.md

```diff
@@ -0,0 +1,335 @@
+Das Spiel befindet sich im [`Dungeon`](https://github.com/Programmiermethoden/Dungeon)-Repo.
+
+Sie benötigen nur dieses Projekt für die Aufgaben, die zusätzlichen Abhängigkeiten werden automatisch über Gradle eingebunden.
+
+*Hinweis: Achten Sie darauf, Daten nur dann in öffentliche Git-Repos zu laden, wenn Sie die nötigen Rechte an diesen Daten haben. Dies gilt insbesondere auch für Artefakte wie Bilder, Bitmaps, Musik oder Soundeffekte.*
+
+## Installation
+
+Für das Dungeon wird das Java Development Kit 17.x.x (JDK 17) oder höher benötigt, stellen Sie sicher, dass Sie es installiert haben.
+
+Laden Sie das Projekt herunter und binden Sie es als Gradle-Projekt in Ihre IDE ein. 
+Eine genauere Anleitung finden Sie [hier](https://github.com/Programmiermethoden/Dungeon/wiki/Import-Project)
+
+Sie können über die run-Funktion Ihrer IDE überprüfen, ob die Installation lauffähig ist. Alternativ können Sie per Konsole in das Dungeon-Verzeichnis wechseln und `./gradlew run` ausführen.
+
+
+*Falls Sie Probleme beim Installieren haben, schauen Sie in die [Kompatibilitätsliste](https://github.com/Programmiermethoden/Dungeon/wiki/JDK-Kompatibilität) und die [FAQ](https://github.com/Programmiermethoden/Dungeon/wiki/FAQ). Melden Sie sich frühzeitig falls Ihr Problem damit nicht behoben werden konnte.*
+
+
+## Grundlagen
+
+Zu Beginn einige grundlegende Prinzipien, die Sie verstanden haben sollten, bevor Sie mit dem Dungeon arbeiten.
+
+Das Dungeon benutzt das Cross-Plattform-Java-Framework [`libGDX`](https://libgdx.com) als Backend.
+Dieses ist im `Dungeon`-Projekt bereits als Abhängigkeit in die Gradle-Konfiguration integriert, Sie müssen dieses nicht extra installieren. Die Ihnen zur Verfügung gestellten Vorgaben sind so umgesetzt, dass Sie kein tieferes Verständnis für `libGDX` benötigen, um die Aufgaben zu lösen. Sollten Sie allerdings einmal auf Probleme stoßen, kann es unter Umständen helfen, einen Blick in die [Dokumentation von `libGDX`](https://libgdx.com/wiki/) zu werfen.
+
+Das `Dungeon`-Projekt fungiert, ganz vereinfacht gesagt, als eine Facade zwischen `libGDX` und Ihrer eigenen Implementierung. Es implementiert ein Entity-Component-System (ECS):
+- Entity: Entitites sind die Elemente (Helden, Monster, Schatzkisten, etc.) des Spiels
+- Component: Components speichern die Datensätze der Entitäten (z.B. die Lebenspunkte)
+- System: Systeme beinhalten die eigentliche Logik und agieren auf die Components
+
+*Weiteres zum ECS im Dungeon erfahren Sie [hier](ecs/ecs_basics.md)*.
+
+Sie selbst nutzen und erweitern die Components und Systeme der Vorgaben. Sie werden ebenfalls neue Entities, Components und Systeme konzeptionieren und implementieren. So erschaffen Sie z.B. Ihre eigenen Monster und fallengespickte Level.
+
+Sie werden im Laufe der Praktika verschiedene Assets benötigen. Diese liegen per Default im `assets`-Verzeichnis. Sie können das Standardverzeichnis in der `build.gradle` anpassen.
+
+- Standardpfad für Texturen: `assets/`
```

Comment:
Die drei Dinger können eigentlich raus.
Das klingt stark nach ("Wenn ich ein Held mache, sucht er automatisch im passenden Ordner", macht er nicht. 
Hab auch keine Ahnung warum wir das damals im QS hatten. Ist eher eine Regel gewesen, damit die assets halbwegs sortiert sind. Mir ist egal wie die Studis ihren asset Ordner sortieren

Reply:
Ich finde es eigentlich nicht schlecht, den Leuten einen Hinweis zu geben, wo sie nach den Texturen suchen können. 

Und ja, mir ist es auch egal, wo sie ihren Kram dann abkippen.

Mir ist nur die Frage nach der Ordnung hochgekommen: Warum haben die Level-Texturen noch einen extra Zwischenordner. Das ist aber keine Frage/Issue für das How-To, sondern eher ein Ticket für @AMatutat. Sorry, dass ich mich da unklar ausgedrückt habe.